aip/aog/3001.md

```diff
@@ -47,80 +47,23 @@ digraph d_front_back {
 
 ### Actions on Google Editors
 
-The Actions on Google editors are the set of people who make decisions on AoG
-AIPs before escalation to the general editors defined in [AIP-1][].
+The Actions on Google editors are the set of people who make decisions on
+Actions on Google AIPs before escalation to the general editors defined in
+[AIP-1][].
 
-The list of AoG AIP editors is currently:
+The list of Actions on Google AIP editors is currently:
 
 - Ali Ibrahim ([@ahahibrahim][])
 - Richard Frankel ([@rofrankel][])
 - Silvano Luciani ([@silvolu][])
 
-The AoG editors have the same responsibilities as the general editors. They
-also have the additional responsibility of establishing correctness of, and
-leadership support for, the contents of AoG AIPs.
+The Actions on Google editors have the same responsibilities as the general
+editors. They also have the additional responsibility of establishing
+correctness of, and leadership support for, the contents of Actions on Google
+AIPs.
 
-AoG AIP editorship is by invitation of the current AoG editors.
```

Comment:
Is deleting this section intentional? If so, can you add info on the change in the CL description.

Reply:
This line (but updated not to have "AoG") is shown below this large deleted block, which is present here because GitHub doesn't (AFAICT) support diffbases. :/

Reply:
Would re-basing your CL against current head fix the issue?

Reply:
Yeah, for some reason I was trying to avoid updating the text I'm about to delete in another PR (or submitting this PR with a bunch of "AoG"s still in the text).  But it doesn't really cost much to do that, so, done.